modules/visualization/dev-instance-foxbox/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing Guidelines
```

Comment:
There is a CONTRIBUTING.md available in the root of ADDF repo. You can delete this here.